pkg/cmd/convert.go

```diff
@@ -62,20 +69,44 @@ func GetValidatedObject(v *viper.Viper, cmd *cobra.Command, out, outerr io.Write
 	}
 
 	var ocObjects []*object.OpenCompose
+	var data []byte
+	var err error
 	for _, file := range files {
```

Comment:
Using filePath makes more sense over using file because it conveys the meaning of what that variable has, here it has path to the file than file itself.